RELEASE.md

```diff
@@ -6,11 +6,11 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-# Release 6.x.x
+# Release 6.0.0
 
 ## Major features and improvements
 
-- Remove metrics plots from metadata panel and add link to the plots on Experiment tracking. (#1268)
+- Remove metrics plots from metadata panel and add link to the plots on experiment tracking. (#1268)
 - Link plot and JSON dataset names from experiment tracking to the flowchart. (#1165)
 - Bump minimum version of React from 16.8.6 to 17.0.2. (#1282)
 - Show preview of data in metadata panel. (#1288)
```

Comment:
I'd consider reordering these so they're more in order of what's interesting to users (e.g. removing plots isn't very interesting, and nor is the react version for most people; adding the preview in is more exciting).

Reply:
The convention is to go from least to greatest PR number. I think we can leave it like this here and reorder by excitement level in the announcement note.